src/attribute.rs

```diff
@@ -1,12 +1,10 @@
 use super::*;
 
-#[derive(
-  EnumString, PartialEq, Debug, Copy, Clone, Serialize, Ord, PartialOrd, Eq, IntoStaticStr,
-)]
+#[derive(EnumString, PartialEq, Debug, Clone, Serialize, Ord, PartialOrd, Eq, IntoStaticStr)]
 #[strum(serialize_all = "kebab-case")]
 #[serde(rename_all = "kebab-case")]
 pub(crate) enum Attribute {
-  Confirm,
+  Confirm(Option<Box<str>>),
```

Comment:
Can this just be `Option<String>`?

Reply:
Yes, I use `Box<str>` to not include the capacity field in the variant. Since this is now by far the largest enum variant, I want to reduce its size. I'll make it an `Option<String>` instead for simplicity.